docs/src/using.md

```diff
@@ -69,7 +70,12 @@ curl localhost:3000/catalog | jq
       "end": 65533
     },
     ...
-  }
+  },
+  "styles": {
+    "americana": {
```

Comment:
TODO(decide) usually paths on the local file system do contain `.json` => possibly remove just this ending or all file suffixes /..

Reply:
removing `.json` is probably enough... what other extensions could it have?

Reply:
no idea what other exstenstions, have removed just `.json` (and spaces)